dokka-templates/README.md

```diff
@@ -0,0 +1,4 @@
+# Customize Dokka's HTML. 
+To customize Dokka's HTML output, place a file in this folder.
```

Comment:
Am I correct that the actual insertion of template and customization happens during the build of kotlin's website on TC? If so, let's describe this process here instead of a generic README to avoid confusion.

Reply:
Thank you, I've updated the readme.